kroxylicious/src/main/java/io/kroxylicious/proxy/config/ConfigParser.java

```diff
@@ -62,6 +64,8 @@ public static ObjectMapper createObjectMapper() {
                 .registerModule(new ParameterNamesModule())
                 .registerModule(new Jdk8Module())
                 .registerModule(new SimpleModule().addSerializer(HostPort.class, new ToStringSerializer()))
+                .registerModule(new SimpleModule().addSerializer(SecretKey.class, new SecretKeySerializer()))
+                .registerModule(new SimpleModule().addDeserializer(SecretKey.class, new SecretKeyDeserializer()))
```

Comment:
Seeing this makes me think about plugins that want to register their own Jackson serializers, but that's a problem for another day.